styles.css

```diff
@@ -76,3 +79,175 @@ a {
   width: 22px;
   padding-right: 15px;
 }
+
+#projects {
+  background-color: #f7f7f9;
+  padding-top: 100px;
+  padding-bottom: 30px;
+}
```

Comment:
Please use `grid` to layout the #project element like asked in the requirements

<img width="1008" alt="image" src="https://user-images.githubusercontent.com/7243132/170837542-cf0f053b-667c-4e38-a987-3a331f1b466b.png">